### DISCUSSION TITLE
Dependency management using Renovate

### DISCUSSION THREAD

**Discussion by shanduur** [s3gw-tech/s3gw#867] [Ideas]
Hi! I suggest adding [Renovate](https://github.com/apps/renovate) to all projects, to improve dependency management. It is [well documented](https://docs.renovatebot.com), free and has a lot of configuration options, supports multiple types of dependencies.

Below sample config that I use in my projects, that probably will be well suited to most of repositories in s3gw organizaion. 

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "automerge": true,
  "extends": [ "config:recommended" ],
  "labels": [ "area/dependency" ],
  "pre-commit": {
    "enabled": true
  },
  "vulnerabilityAlerts": {
    "enabled": true
  },
}
```

**Comment by jecluis** [comment]
Hi @shanduur - thank you for the suggestion.

I have created #868 and we'll prioritize it once we have our next release, unless we find someone willing to work on it :)
